src/NautilusRefactoring/GlobalVariable.extension.st

```diff
@@ -1,6 +1,6 @@
 Extension { #name : #GlobalVariable }
 
 { #category : #'*NautilusRefactoring' }
-GlobalVariable >> executeRenameRefactoringBy: aNautilusRefactoring inClass: aClass [
-	aNautilusRefactoring renameClass: value 
+GlobalVariable >> executeRenameRefactoringBy: aNautilusRefactoring [
+	aNautilusRefactoring renameClass: self definingClass 
```

Comment:
It looks like a bug. There is no definingClass for globals

Reply:
Probably because there are no tests about this.

Reply:
yes, but if we require 100% perfect tests before doing anything, we should stop now